runtime/src/config.rs

```diff
@@ -26,6 +26,9 @@ use std::path::PathBuf;
 use std::collections::HashMap;
 use std::collections::HashSet;
 
+// 65536*1600 ~ 100 Mb
```

Comment:
Maybe it will be good to write in comment that page size = 64Kb

Reply:
Ok, added